.changelog/2796.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
should this be `release-note:bug` or feature?
```suggestion
```release-note:bug
```

Reply:
Yeah I went back and forth on this one. Will ask Jeff